src/main/app/forms/models/itemInMultiRowForm.ts

```diff
@@ -0,0 +1,6 @@
+export abstract class ItemInMultiRowForm {
```

Comment:
`MultiRowFormItem`?

Reply:
Or `FormRow`?

Reply:
I'd rather call it `MultiRowFormItem`. More descriptive imo.